lib/txdb/backends/globalize/reader.rb

```diff
@@ -12,11 +14,42 @@ def initialize(table)
         end
 
         def read_content
-          { origin_table_name(table.name) => content_for_records }
+          [Txdb::TxResource.new(resource, serialized_content)]
         end
 
         private
 
+        def resource
+          @resource ||= Txgh::TxResource.new(
```

Comment:
Yeah, this is correct. Txdb and Txgh both define the concept of a resource. `Txdb::TxResource` actually wraps `Txgh::TxResource` together with some content.